### PR TITLE
User detail everywhere

### PR DESCRIPTION
**Pull request by alinhle**
## What does this PR do?
- Clicking on username anywhere (on the admin side) will open the user detail drawer.

## How to test this PR
- Click on the username in a comment, in the the flag box, in a flagged user, or in people on the community page should trigger the user detail drawer.

### CODE REVIEW

File: client/coral-admin/src/components/CommentAnimatedEdit.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        enterActive: styles.bodyEnterActive,
+        leave: styles.bodyLeave,
+        leaveActive: styles.bodyLeaveActive,
+      }}
+      transitionEnter={true}
+      transitionLeave={true}
+      transitionEnterTimeout={3600}
+      transitionLeaveTimeout={2800}
+    >
+      {React.cloneElement(React.Children.only(children), {key: murmur3(body)})}
```
Comment (okbel) [inline client/coral-admin/src/components/CommentAnimatedEdit.js:21]: Interesting!

File: client/coral-admin/src/reducers/moderation.js [hunk 2]
```diff
@@ -1,13 +1,9 @@
-import {fromJS, Set} from 'immutable';
+import {fromJS} from 'immutable';
```
Comment (okbel) [inline client/coral-admin/src/reducers/moderation.js:1]: We're one step closer from removing Immutable haha

**Comment by okbel** [comment]
Perfect! 💯 

We will need a new design pass in the near future. (All the usernames in the admin side are clickable - maybe we need to add some style here and there for it) /cc @IAmSamHankins

**Review by okbel — APPROVED** [review]

**Comment by alinhle** [comment]
> We will need a new design pass in the near future. (All the usernames in the admin side are clickable - maybe we need to add some style here and there for it) /cc @IAmSamHankins

Yes I thought about that. Then the styles need to be harmonized. I skipped this because PR was already getting big :-)

**Pull request merged**